deploy/staging/cronjob.yaml

```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: refresh-views
 spec:
-  schedule: "0 0 * * *"
+  schedule: "5 1 * * *"
```

Comment:
I'm not sure I understand how changing the time the job fires fixes the issue? 😢

Reply:
Sorry - this was part of my initial thinking, but will change back now I've realised it's indentation yaml pain